easybuild/easyconfigs/x/xorg-macros/xorg-macros-1.19.2-GCCcore-8.2.0.eb

```diff
@@ -1,16 +1,26 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::
+#
+# Notes::
+##
+
 easyblock = 'ConfigureMake'
 
 name = 'xorg-macros'
 version = '1.19.2'
 
-homepage = 'http://cgit.freedesktop.org/xorg/util/macros'
+homepage = 'https://cgit.freedesktop.org/xorg/util/macros'
 description = """X.org macros utilities."""
 
 toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
 
-source_urls = ['http://cgit.freedesktop.org/xorg/util/macros/snapshot']  # no slash ('/') at the end!
-sources = ['util-macros-%(version)s.tar.gz']
-checksums = ['5d6e62803c9f20830b28ad8a9800c97a9987635089229e3e380ef3848c84d2a6']
+source_urls = ['https://gitlab.freedesktop.org/xorg/util/macros/-/archive/util-macros-%(version)s']
+sources = ['macros-util-macros-%(version)s.tar.gz']
+checksums = ['326e51a5d673e3d9cc7f139aee469a11e18ea060b1d06c22694612e68a2089b1']
```

Comment:
Hmm, the original source tarball is not available anymore, from anywhere? :(

Reply:
Hmm, it seems they have alternative tarball available for other versions, do @boegel still want me to update all url in the folder. Checksums will all need to be updated though, see what you reckon.

Reply:
Not I am aware of though. /eyes